Hobbify-Stats/querys.py

```diff
@@ -0,0 +1,91 @@
+import postgres
```

Comment:
All functions (except for `insertScore`) share the exact same code. This can be heavily reduced by using a different program structure while maintaining a *method-per-query* schema. 

You might also consider a using the singleton pattern here:

```python
class Storage:

  queries = {
     'pending': """SELCT COUNT(*)...""",
     'done': """SELCT COUNT(*)...""",
     ...
  }

   def __init__():
       self.conn = pstgrs.connect() 

  def run_count_query(self, query, data):
    cur = self.conn.cursor()
    cur.execute(query, data)
    return cur.fetchone()

   def pending_instances(self, username):
      return self.run_count_query(self.queries['pending'], [username])

   def done_instances(self, username):
      return self.run_count_query(self.queries['done'], [username])

   ...
```

Although you must be careful about having a single connection, you can read more about postgresql connection pool and trheard-safe objetcs in`psycopg2`'s documentation.